river/anomaly/sad.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+from river import anomaly, stats
+
+__all__ = ["StandardAbsoluteDeviation"]
+
+
+class StandardAbsoluteDeviation(anomaly.base.AnomalyDetector):
```

Comment:
The issue with this method is that it is univariate. You're trying to fit into the multivariate anomaly detection framework, which is not the right way.

What you should do is inspire yourself from `anomaly.GaussianScorer`. It's also a univariate anomaly detection method.

Let me know if this isn't clear

Reply:
@MaxHalford I think I can understand the idea, and have also implemented it in my upcoming commits. It would be much appreciated if you can have a look through and see whether the new approach is appropriate!